profile/README.md

```diff
@@ -64,6 +64,12 @@ As a consequence of all that, the only sure way to ensure some issue gets fixed
 
 As you contribute more and more, you will eventually get nominated as a committer on the project.
 
+## Contact
+
+If you have questions about contributions or would like to get in touch with the developers of the Eclipse IDE, join the [Eclipse Foundation chat](https://chat.eclipse.org/docs/) and visit the rooms
+- [Eclipse IDE](https://chat.eclipse.org/#/room/#eclipse-ide-general:matrix.eclipse.org)
+- [Eclipse IDE Newcommers](https://chat.eclipse.org/#/room/#eclipse-ide-newcomers:matrix.eclipse.org)
+
```

Comment:
After a discussion with ed, discussions would be better rather than issues ... I will open it via otterdog.  Then, if a discussion implies to open issue (either in eclipse-ide or in other projects), we will advise.

Reply:
BTW, we could also have a button in the readme page 'Ask for Assistance' that will go to a dedicated page with the matrix links and explanations to use the discussions... WDYT ?

Reply:
I would prefer to have a single page for assistance, that can change independently and with a button like this : 

![assistance](https://img.shields.io/badge/Ask_for_assistance-red?style=for-the-badge&logo=eclipseide)

to access from several places ... we could add this button in any page to drive the users to the assistance if needed...  

On the assistance page we could have : the matrix links (with an explanation how to use it (element...), and the discussion link to search in existing discussions or to open an new one...

Reply:
Discussions are opened here : https://github.com/eclipse-ide/.github/discussions

There is category Q&A to get the feedback... I think this is the right place for assistance... better than issues...

Reply:
I will move your text in a specific page and re-push  it in this PR...

Reply:
> I would prefer to have a single page for assistance, that can change independently and with a button like this :
> 
> ![assistance](https://camo.githubusercontent.com/5aefcd23edd5a9883d55dc406e772b82fe2949eca2e52bf0e519c127a2e06cfa/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f41736b5f666f725f617373697374616e63652d7265643f7374796c653d666f722d7468652d6261646765266c6f676f3d65636c69707365696465)
> 
> to access from several places ... we could add this button in any page to drive the users to the assistance if needed...

Yes that sounds reasonable.

> On the assistance page we could have : the matrix links (with an explanation how to use it (element...), and the discussion link to search in existing discussions or to open an new one...

Yes, but at the same time I think we should revise our strategy here a bit. To be sustainable with our resources the goal should be that the documentation in this repository is sufficient for new contributors to complete their contribution without our assistance. Our 'assistance' should only be necessary to review PRs and to discuss the technical implementation, but all the repeating basics should be doable only from the provided documentation.